lib/active_support_type_extensions/array.rb

```diff
@@ -11,18 +11,10 @@ def cast_value(value)
       when String
         value.split(",")
       when ::Array
-        value.map { |item| convert_to_integer(item) }
+        value.map { |item| Integer(item, exception: false) || item }
```

Comment:
I re-read that code while I was working on this PR, it’s just a very small refactor to remove unnecessary code.